### DISCUSSION TITLE
Double Vpn

### DISCUSSION THREAD

**Discussion by YRS200** [angristan/openvpn-install#1079] [Help]
hi guys i have something on my mind i want to double vpn

**Comment by YRS200** [comment]
example: i will have 2 servers one of them is working server in my country 2nd server is blocked server in my country first it needs to connect to 1 st server working server after that need to connect 2nd server not working server, not working server need to connect through working server but when i check ip location it needs to show me 2nd servers ip(blocked server) how to do it if someone know's it can you tell me plz!

**Comment by nimamoradi** [comment]
@YRS200 tell me if you found somthing
